README.md

```diff
@@ -33,7 +70,7 @@ See [Developer's Guide](https://docs.celo.org/) for full details about the desig
 
 ## Issues
 
-See the [issue backlog](https://github.com/celo-org/celo-monorepo/issues) for a list of active or proposed tasks. Feel free to create new issues to report bugs and/or request features. Please add labels to your issues, tagging the appropriate package/area.
+See the [issue backlog](https://github.com/celo-org/celo-monorepo/issues) for a list of active or proposed tasks. Feel free to create new issues to report bugs and/or request features.
```

Comment:
"non-collaborators" are not able to add labels I think.
Maybe the Issue template could have categories?

Reply:
See #2111